cloudflare/resource_cloudflare_page_rule.go

```diff
@@ -745,12 +745,12 @@ func transformFromCloudflarePageRuleAction(pageRuleAction *cloudflare.PageRuleAc
 					fieldOutput[fieldID] = fieldValue
 				}
 
-				if fieldOutput["exclude"] == "*" {
-					fieldOutput["exclude"] = []interface{}{}
+				if itemExistsInSlice(fieldOutput["exclude"], "*") {
+					fieldOutput["exclude"] = []interface{}{"*"}
```

Comment:
We should not add `*` here. It was left out on purpose, so the `ignore` field has its semantics.

Reply:
This was added as the API throws an error when `query_string {}` is passed without any options (_something_ is required). Should we instead use all query string parameters as the default?

Reply:
My apologies, I thought this was transforming from API to Terraform, not the other way around.